pkg/performanceprofile/controller/performanceprofile_controller_hypershift.go

```diff
@@ -17,20 +17,51 @@ limitations under the License.
 package controller
 
 import (
+	"bytes"
+	"context"
 	"fmt"
+	"hash/fnv"
+	"strings"
 
+	performancev2 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v2"
+	tunedv1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1"
 	ntconfig "github.com/openshift/cluster-node-tuning-operator/pkg/config"
+	"github.com/openshift/cluster-node-tuning-operator/pkg/performanceprofile/controller/performanceprofile/components/manifestset"
 
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/klog"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	serializer "k8s.io/apimachinery/pkg/runtime/serializer/json"
+	"k8s.io/klog/v2"
 
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-const controllerGeneratedMachineConfig = "hypershift.openshift.io/performanceprofile-config"
+// REVIEW - Most of these already are defined in 'pkg/operator/hypershift.go' maybe we should avoid duplication.
+//
+//	What about creating a file with all these constants and labels and reference them both here and in the
+//	NTO hypershift code?
+//
+// REVIEW - Reorder constants to read them better.
```

Comment:
this is likely a good idea, let's do in a later PR as followup

Reply:
issue created: https://github.com/openshift/cluster-node-tuning-operator/issues/859